src/routes/routes.tsx

```diff
@@ -19,12 +19,14 @@ import RootWithQuery from './root-with-query';
 import InngangKategorier from './inngang/inngang-kategorier';
 import InngangInnsendingDigital from './inngang/inngang-innsendingsvalg-digital';
 import InngangInnsendingPost from './inngang/inngang-innsendingsvalg-post';
+import FormTitle from './form-title';
 
 const App = () => (
     <React.StrictMode>
         <BrowserRouter>
             <AppContextComponenet>
                 <MainContainer>
+                    <FormTitle />
```

Comment:
Siden denne kun skal vises for klager, så kan vi sette den i `FormContainer`.
Da slipper vi hente klagen fra `context`, å `null`-checke den og vi slipper å rendre den uten en klage i `Inngangen`.

Reply:
Jeg har testet dette lokalt, for å være sikker på at det går.
Om du er enig i at det er en forbedring kan jeg godt pushe det.